### PR TITLE
Rb/keycloak auth template

### PR DESCRIPTION
**Pull request by RohitBhosale204**

### CODE REVIEW

**Review by sanketcalfus — APPROVED** [review]

File: express/AppConfig.ts.liquid [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export const AppConfig = {
+  AUTH_SERVICE_URL: 'http://localhost:3002',
```
Comment (salim-jujara) [inline express/AppConfig.ts.liquid]: Why localhost:3002? How this will work on AWS or any other cloud?

File: keycloak/config.sh.liquid [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+ 
+##Third curl command to create client
+curl -k --silent --show-error --request POST \
+--url "${KEYCLOAK_BASEURL}/admin/realms/${REALM_NAME}/clients" \
+--header "Content-Type: application/json" \
+--header "Authorization: Bearer ${token}" \
+--data-raw "$CLIENT_PAYLOAD"
+ 
+# Fourth curl command to create user
+emails=('frances@example.com' 'taylor@example.com' 'ben@example.com' 'zachary@example.com' 'lily@example.com' 'rosea@example.com' 'daisyb@example.com' 'johng@example.com')
```
Comment (salim-jujara) [inline keycloak/config.sh.liquid]: Create just one user for demo with some generic name and default password. We don't need to create 7 users

File: keycloak/deployment.yml.liquid
Comment (salim-jujara) [inline keycloak/deployment.yml.liquid:1]: Deployment.yaml should be part of gitops repo and not here..

File: keycloak/docker-compose.yml.liquid
Comment (salim-jujara) [inline keycloak/docker-compose.yml.liquid:1]: Why do we need docker-compose file? If at all we need to test magikube locally, we should think about setting up everything on minikube (devops help needed) I will add ticket for this

**Review by cloudops-calfus — APPROVED** [review]

**Review by salim-jujara — APPROVED** [review]

**Review by adi112g — APPROVED** [review]

**Pull request merged**